tokenomics/elixir.md

```diff
@@ -16,9 +16,7 @@ SubDAOs use locked Elixir to burn a small percentage of their governance token e
 
 ## Elixir Development
 
-In the first version, Elixir will be a simple Balancer pool that consists of one third Dai, one third Lido stETH and one third MKR. This initial implementation has no dependency on EtherDAI which speeds up deployment. Later, the Balancer pool will be modified to replace stETH with EtherDAI.
+In the first version, Elixir will be Uniswap V2 DAI/MKR liquidity pool tokens.
 
```

Comment:
Since there's no final version of Elixir, should we drop the "In the first version" part? Or alternatively state something like "Future versions of Elixir may include ETH or EtherDAI in the liquidity pool", if that's still on the cards